NSS-Day-Cohort-53/hello-world-mashed-potatoes
# created main.css for general styling and imported other css files int…

…o it

Steps to test:

- load site in browser
- make sure all css works the same as before for each page
